README.md

```diff
@@ -36,7 +36,11 @@ To install `poutine`, download the latest release from the [releases page](https
 
 #### Homebrew
 ``` bash
-brew install boostsecurityio/tap/poutine
+[~] brew verify poutine
```

Comment:
what is this command? it's not in the [man page](https://docs.brew.sh/Manpage) and i'm getting "Error: Unknown command: verify"

Reply:
Same @fproulx-boostsecurity do you have a preview version of homebrew?

Reply:
https://github.com/trailofbits/homebrew-brew-verify
it's a plugin at the moment, I'll remove it